packages/app-store/routing-forms/pages/route-builder/[...appPages].tsx

```diff
@@ -233,6 +264,25 @@ const Route = ({
   }
 
   const shouldShowFormFieldsQueryBuilder = (route.isFallback && hasRules(route)) || !route.isFallback;
+  const eventTypeRedirectUrlOptions =
+    eventOptions.length !== 0
+      ? [{ label: t("custom"), value: "custom", eventTypeId: 0 }].concat(eventOptions)
+      : [];
+
+  const eventTypeRedirectUrlSelectedOption =
+    eventOptions.length !== 0 && route.action.value !== ""
+      ? eventOptions.find(
+          (eventOption) => eventOption.value === route.action.value && !customEventTypeSlug.length
+        ) || {
+          label: t("custom"),
+          value: "custom",
+          eventTypeId: 0,
+        }
+      : undefined;
```

Comment:
Nothing new here. There are Select `options` and `value` props respectively for the EventType Redirect URL Select